src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -750,11 +856,71 @@ ActStatus ActionHandlers::cs_req_to_sgw(SM::ControlBlock& cb)
 	const DigitRegister15& ueMSISDN = ue_ctxt->getMsisdn();
 	ueMSISDN.convertToBcdArray(cs_msg.MSISDN);
 
-	cmn::ipc::IpcAddress destAddr;
-	destAddr.u32 = TipcServiceInstance::s11AppInstanceNum_c;
+	//cmn::ipc::IpcAddress destAddr;
+	destAddr_dns.u32 = TipcServiceInstance::s11AppInstanceNum_c;
```

Comment:
not sure what was purpose of changing this variable name.. ? Dns seems mis leading, since we are dispatching message to s11 instance.